insights/tests/__init__.py

```diff
@@ -154,11 +163,24 @@ def run_test(component, input_data,
         do_filter: Does need to check dependency spec filter warning?
             - it's not required to check the filters for sosreport
     """
+    def get_filtered_specs(module):
+        filtered = set()
+        mods = dir(importlib.import_module(module))
+        for comp, parsers in COMPONENT_FILTERED_PARSERS.items():
+            if comp in mods:
+                for parser in parsers:
+                    if parser.split('.')[-1] not in mods:
+                        # The parser is NOT imported again in the rule
+                        parser = dr.get_component_by_name(parser)
+                        filtered.update(dr.get_registry_points(parser))
+        return filtered
+
     if do_filter and filters.ENABLED:
-        mod = component.__module__
-        sup_mod = '.'.join(mod.split('.')[:-1])
+        mod = dr.get_module_name(component)
+        sup_mod = dr.get_base_module_name(component)
```

Comment:
Hi @xiangce , for this `sup_mod` getting line,  I see the new code is mean to get the mod itself instead of the it's `sup`. For example, 
```
mod:  insights.plugins.combiner_rule_wo_filterable_parser
sup_mod (in old code):  insights.plugins
sup_mod (in new code):  combiner_rule_wo_filterable_parser
```
Do you mean such changes?

Reply:
Good catching! Thanks @JoySnow - it's fixed in the new commit.